src/applications/vaos/actions/expressCare.js

```diff
@@ -76,21 +77,26 @@ export function updateFormData(page, uiSchema, data) {
   };
 }
 
-export function openReasonForRequestPage(page, uiSchema, schema) {
+export function openAdditionalDetailsPage(page, uiSchema, schema, router) {
   return (dispatch, getState) => {
     const state = getState();
-    const email = selectVet360EmailAddress(state);
-    const homePhone = selectVet360HomePhoneString(state);
-    const mobilePhone = selectVet360MobilePhoneString(state);
-    const phoneNumber = mobilePhone || homePhone;
-    dispatch({
-      type: FORM_REASON_FOR_REQUEST_PAGE_OPENED,
-      page,
-      uiSchema,
-      schema,
-      email,
-      phoneNumber,
-    });
+    const data = selectExpressCareFormData(state);
+    if (!data.reason) {
```

Comment:
If user happens to go directly to this page, or hits back on the confirmation page, redirect them back to the info page.

Reply:
👍🏼 Moved this redirect logic to `<NewExpressCareRequestLayout/>`